dashboard/app/models/artist.rb

```diff
@@ -264,4 +275,44 @@ def k1_blocks
       <block type="simple_jump_left"></block>
     XML
   end
+
+  def strip_block(block)
+    stripped_block = block.dup
+    stripped_block.xpath(STRIPPED_NODES_XPATH).remove
+    stripped_block['type'] = stripped_block['type'].chomp '_dropdown'
```

Comment:
We should consult with Ryan; one of the issues we'd like to address in the longer term is exactly that kind of discrepancy between what's in the solution and what's in the toolbox.

Most notably, we've had problems with required/recommended blocks pushing the wrong "kind" of block; we'd like to ultimately use solutions to validate that the required/recommended blocks are a subset of solution and that solution is a subset of toolbox.

However, I think that this is more than fine for now, and based on how it's built it'll be straightforward to update if we decide to!